src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER = 100;
+    private static final Random RANDOM = new Random();
+
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(),
+                RANDOM.nextInt(MAX_NUMBER + 1));
```

Comment:
just curious why `+ 1`?

Reply:
nextInt(int bound) method returns:
the next pseudo-random, uniformly distributed int value between zero (inclusive) and boundary (EXCLUSIVE) from this random number generator's sequence

Reply:
i add public getRandomNumberFromRange, returns random int in range [firstInteger, secondInteger] which inclusive this two integer